packages/playground/src/weblets/tf_funkwhale.vue

```diff
@@ -26,24 +26,7 @@
         #="{ props }"
       >
         <input-tooltip tooltip="Instance name.">
-          <v-text-field label="Name" v-model="name" v-bind="props" />
-        </input-tooltip>
-      </input-validator>
-
-      <input-validator
-        :value="username"
-        :rules="[
-          validators.required('Username is required.'),
-          validators.isLowercase('Username should consist of lowercase letters only.'),
-          validators.isAlphanumeric('Username should consist of letters and numbers only.'),
-          (username: string) => validators.isAlpha('Username must start with alphabet char.')(username[0]),
-          validators.minLength('Username must be at least 2 characters.', 2),
-          validators.maxLength('Username cannot exceed 50 characters.', 50),
-        ]"
-        #="{ props }"
-      >
-        <input-tooltip tooltip="Funkwhale admin username.">
-          <v-text-field label="Username" v-model="username" v-bind="props" />
+          <v-text-field label="Name/Username" v-model="name" v-bind="props" />
```

Comment:
i think this should be only the name, and we can add a description to the tooltip, like `instance name, will be used as a username`

Reply:
![image](https://github.com/user-attachments/assets/e8fcd19f-fbd4-4c84-9ff6-4cffdeaff720)
Email can do it too, @AhmedHanafy725, so can Email do it instead of the deployment name?